assignment-1/Failed__case.md

```diff
@@ -1,87 +1,86 @@
+
+#####Timeline of problems and solutions.
+
+######Before the launch
+There were a lot of warnings before the lauch of the website by different parties. But is seems nothing is done with them. Below are a couple of them noted
+
+On April 4, 2013 a 15-page document by McKinsey & Co. warns among others the CMS administrator that there was insufficient end-to-end testing and advises that a limited initial launch of the website would be ideal. It seems nothing is done with this report, because it surfaced only half November 2013. One and a half month after the lauch. Next to that there were only two weeks of end-to-end testing before the launch took place for all the users.
+
+The Government Accountability Office (GAO), an independent organisation of the government, reported in June 2013 that there could be trouble with the goal of launching the website on the first of October 2013. 
+
+On 10 September there was a panel of the House of Representatives who were hearing among others consulting firm Leavitt Partners and contractor CGI. CGI stated during this hearing that it was on schedule for launching the website. On the other hand Leavitt Partners stated that the launch will be “rocky” [4, page 7] because there are still ongoing technology challenges for the online health insurance exchange.
+
+According to the New York Times [4, page 7], documents released by House investigators reveal that the ‘testing bulletin’ showed that the website as of September 30 could handle only about 1100 users at a time, even though officials had said it should have been abble to accomodate perhaps as many as 60,000 users.
+
+######After the launch
+During its launch (1 October 2013) the website did not perform well. Only 271,000 of the 9.47 million users that try to registrate where succesfull. ?There was already $700 million dollar spend? In 2014 the total number of costs of the project is 1.7 billion dollar (reference). Intuitively I ask myself what would have happened if they had established a schedule of access to the site?. There were official warnings telling that the site was going to be unable to accommodate so many users at once.
+
+An extra team is recluted (22th October 2013) It was recognized that product testing was insufficient. Serious underestimation of time.
+
+QSSI is selected as the contractor responsible to now oversee federal website fixes (October 27, 2013). A single data center brings down the mission-critical website and should be presumed and avoided. Such requirements should be identiﬁed and addressed before project completion.
+
+(October 31, 201) Again an unrealistic schedule, site crash for second time due small amount of servers, an explicit gap at project definition and planning.
```

Comment:
The year 201 is indeed very unrealistic! ;-)
Furthermore; Site crashed. (Should be a new sentence as well)